Makefile

```
@@ -153,6 +156,20 @@ bmalph-init: ## Initialize BMALPH for current project; set BMALPH_DRY_RUN=true t
 bmalph-setup: ## Install and initialize BMALPH for current project; defaults to BMALPH_PLATFORM=codex
 	@$(MAKE) bmalph-init BMALPH_PLATFORM="$(BMALPH_PLATFORM)" BMALPH_DRY_RUN="$(BMALPH_DRY_RUN)"
 
+docs: docs-check docs-api ## Validate docs and generate source API reference into build/docs
```

checkmake (0.3.2)

[warning] Target "docs" should be declared PHONY.

(phonydeclared)